yarn.lock

```diff
@@ -4125,7 +4132,7 @@ tiny-relative-date@^1.3.0:
   resolved "https://registry.npmjs.org/tiny-relative-date/-/tiny-relative-date-1.3.0.tgz"
   integrity sha512-MOQHpzllWxDCHHaDno30hhLfbouoYlOI8YlMNtvKe1zXbjEVhbcEovQxvZrPvtiYW630GQDoMMarCnjfyfHA+A==
 
-tinycolor2@^1.4.1:
+tinycolor2@^1.4.1, tinycolor2@^1.4.2:
```

Comment:
tinycolor2@^1.4.1 ? probably some kind of residue ?

Reply:
style-dictionary uses the `1.4.1` version:
```
style-dictionary@^3.0.1:
    [...]
    tinycolor2 "^1.4.1"
```
and omod and design-tokens library are using `1.4.2` version.